package.json

```diff
@@ -2327,6 +2337,11 @@
 					"group": "inline@2",
 					"when": "view == notifications:github && (viewItem == 'Issue' || viewItem == 'PullRequest')"
 				},
+				{
+					"command": "notification.markAsRead",
+					"group": "inline@3",
```

Comment:
In general, we try to make sure any action that's added `inline` is also added to the right click context menu.

Reply:
Hi I am not quite sure I understand. Do you mean I should add the command `notification.markAsRead` elsewhere too?

Reply:
We have the inline actions (circled), and the right click actions (squared). If an action is in the inline actions, we usually also put it in the right click actions.

![image](https://github.com/user-attachments/assets/23849c3a-0a30-4b23-9b5c-d212135b25b6)

Reply:
I see better thanks. I added the actions into the group `issues_0@<number>` which placed them into the right-click context menu